vms/pom/tests/tests.md

```diff
@@ -0,0 +1,143 @@
+The tests which are needed by POM app have already been tested thus following _DRY_ approach, the tests which are needed
+by POM app have been documented below. 
```

Comment:
Is there a reason for this line change?

Reply:
No, its just me that I tend to break line if it goes out of pycharm box. Should I change it?

Reply:
Yeah, let markdown wrap text on its own. Otherwise we might get disjointed sentences on different screen sizes. I saw this in a couple of other places as well.

Reply:
Okay, on it.